tests/test_graphql_client.py

```diff
@@ -83,13 +83,21 @@ def test_raises_http_errors_as_exceptions(self, post_mock):
     @patch("python_graphql_client.graphql_client.requests.post")
     def test_execute_query_with_headers(self, post_mock):
         """Sends a graphql POST request with headers."""
-        headers = {"Content-Type": "application/json"}
-        client = GraphqlClient(endpoint="http://www.test-api.com/", headers=headers)
+        client = GraphqlClient(
+            endpoint="http://www.test-api.com/",
+            headers={"Content-Type": "application/json", "Existing": "123",},
+        )
         query = ""
-        client.execute(query)
+        client.execute(query=query, headers={"Existing": "123", "New": "foo"})
```

Comment:
I think the value for the `Existing` key here should be different than the original value. Changing the value to something like `"456"` would make sure that the tests can't give a false positive.

Reply:
Oh, shit! I used to be doing that but while trying to making this test look better, I screwed that up somehow. 😆